### ISSUE TITLE
Landing Page interface

### ISSUE THREAD

**Issue by Ngo-Bassom-Anne-Rosa** [PacomeKFP/snappy#2]
![Screenshot from 2024-12-18 00-55-08](https://github.com/user-attachments/assets/765b187c-b8e4-43ea-9483-98e9c6ed3ed5)
This is the first page that the users when he enters the app, for the dashboard side, we can simply a logo of the app and change the policy that is used. Also, the first line should be the differents projets of the owner,  with a small description of the project with an image of the projects. I think a search bar, is not necessary for the first page, we can remove it

**Comment by PacomeKFP** [comment]
So here we remove graphs and stats

is it ?

**Comment by Ngo-Bassom-Anne-Rosa** [comment]
yes, there are not useful for me, after we can add them just to show the usage of the projects

**Comment by PacomeKFP** [comment]
I was thinking about displing simple stats as: Nomber of users, Nomber of messages, Average of messages per day and other metrics

Knowing that we are building a dashboard
not a chat app
